mlibrary/arclight
# tune event handlers for turbolinks

The event handler added by `DOMContentLoaded` gets unloaded; add the checkbox handlers during `turbolinks:load` instead.

How to test:

* open http://localhost:3000/catalog/umich-bhl-032_aspace_5c2c579052729770ae58706e6cc853df
* click a checkbox
* click the "Request" button
* a new tab to aeon should open
